specs/data_structures.md

```diff
@@ -170,6 +172,24 @@ HashDigest is a [type alias](#type-aliases).
 
 Output of the [hashing](#hashing) function. Exactly 256 bits (32 bytes) long.
 
+### FeeHeader
+
+| name       | type     | description                                      |
+| ---------- | -------- | ------------------------------------------------ |
+| `baseRate` | `uint64` | The base fee rate for this block.                |
```

Comment:
Why not call it baseFee if it is the base fee?

Reply:
Because it's not a fee, it's a rate (per byte). Similarly to how "gas price" isn't a fee, but a rate.